101-pubnub-eventhub-bridge/azuredeploy.json

```diff
@@ -0,0 +1,312 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "Event Hub Namespace": {
+      "type": "String",
+      "defaultValue": "pn-eventhub-1fba",
+      "metadata": {
+        "description": "A unique Namespace. It's suggested to replace the 'pn-' prefix with your own unique prefix, such as your company name, and add a '-suffix' at the end, where the suffix is also unique."
+      }
+    },
+    "Azure Webjob Name": {
+      "type": "String",
+      "defaultValue": "pn-webjob-80fd",
+      "metadata": {
+        "description": "A unique Web Job Name. Follow the same naming conventions as for Event Hub Namespaces to ensure you have a unique, legal string."
+      }
+    },
+    "Azure Datacenter Location": {
+      "type": "String",
+      "defaultValue": "westus",
+      "metadata": {
+        "description": "Must be westus unless you ensure all components are in the same region.  See README for more information."
```

Comment:
A TODO for me is to make this value adjustable, I'd prefer to keep it as-is for now until I square it away in the finalized "user-defined-region" version.